src/Sylius/Component/Promotion/Generator/CouponGenerator.php

```diff
@@ -43,16 +43,21 @@ public function __construct(RepositoryInterface $repository, EntityManagerInterf
      */
```

Comment:
You'll need to change the doc either here or in the Interface file for the @return param